lib/modules_long.txt

```diff
@@ -0,0 +1,83 @@
+audit-tools/audit-stream.js:var assert = require('assert');
```

Comment:
what is the purpose of this file and how does it get generated and should it be committed?